### ISSUE TITLE
WORKDIR should allow env variables

### ISSUE THREAD

**Issue by gudleik** [RedCoolBeans/dockerlint#17]
### Issue details

Using an environment variable as WORKDIR should be allowed.
### Steps to reproduce/test case

This is valid syntax in a Dockerfile:

``` Dockerfile
ENV MY_PATH=/var/foo
WORKDIR $MY_PATH
```

But dockerlint says `WORKDIR path $MY_PATH must be absolute`.
### Affected version(s)
- linter-docker v0.1.2

**Comment by SharpEdgeMarshall** [comment]
Duplicate of #9
